src/topper-web-sdk.ts

```diff
@@ -18,13 +18,39 @@ class TopperWebSdk {
       environment: Environments.PRODUCTION,
       is_android_app: false,
       is_ios_app: false,
+      use_assets: false,
```

Comment:
or make this an array `['assets']`, default to `[]` and append on [116](https://github.com/uphold/topper-web-sdk/pull/2/files#diff-ef7abfd1514da83536fd8baee5ee35e6ab88ce9672304c26eb6e5c8a11abe7a2R116) accordingly.